TestProject/TestProject/Views/ListItemCell.swift

```diff
@@ -7,6 +7,14 @@
 
 import UIKit
 
+protocol ListItemCellProtocols
+{
+    func didTapChecked()
+}
+
+
+
+
```

Comment:
You can remove `override func setSelected(_ selected: Bool, animated: Bool)` method.